src/system/PageLockModule/tables.php

```diff
@@ -1,46 +0,0 @@
-<?php
```

Comment:
I have hesitated to delete `tables.php` files because doing so in a previous refactor (search module), the files was needed for BC. I don't know when this is the case, so I have simply left them. thoughts?